### PR TITLE
Error fixed: Couldn't run the command locally

### PR DESCRIPTION
**Pull request by aemonge**
Fixed the following error:
```
fs.js:646
  return binding.open(pathModule._makeLong(path), stringToFlags(flags), mode);
                 ^

Error: ENOENT: no such file or directory, open './usage.txt'
    at Object.fs.openSync (fs.js:646:18)
    at Object.fs.readFileSync (fs.js:551:33)
    at Object.<anonymous> (/home/aemonge/usr/docs/cells-hybrid-solution/node_modules/github-mar
n/cmd.js:30:27)
    at Module._compile (module.js:643:30)
    at Object.Module._extensions..js (module.js:654:10)
    at Module.load (module.js:556:32)
    at tryModuleLoad (module.js:499:12)
    at Function.Module._load (module.js:491:3)
    at Function.Module.runMain (module.js:684:10)
    at startup (bootstrap_node.js:187:16)

```

### CODE REVIEW

File: cmd.js [hunk 1]
```diff
@@ -21,13 +22,14 @@ const argv = minimist(process.argv.slice(2), {
 });
 
 if (argv.v || argv.version) {
-  process.stdout.write(require('./package').version + '\n');
+  process.stdout.write(require(`${__appRoot}/package`).version + '\n');
```
Comment (1000ch) [inline cmd.js]: Just use `__dirname`?

Comment (aemonge) [reply cmd.js]: let me try it :)

**Comment by aemonge** [comment]
Made the requested changes, also fixed the .pug template to use a basedir instead of the conditional statements which failed.

**Comment by aemonge** [comment]
Can we run travis again ?   I should have fixed the `npm test`

**Pull request closed without merging**